app/boards/shields/petejohanson_proton_handwire/keymap/keymap.c

```diff
@@ -10,22 +10,22 @@ bool zmk_handle_key_user(struct zmk_key_event *key_event)
 	case CC_LOWR:
 		if (key_event->pressed)
 		{
-			zmk_keymap_layer_activate(1);
+			zmk_keymap_layer_activate(LOWER);
```

Comment:
How do we plan to abstract layer state/changes when dealing with runtime keymap/layer changes?

Reply:
This is probably pure configuration moving forward. In our discussions on Discord instead of this custom code, this would instead be something like an action assigned to this position with:

* Action Type: Momentarily Activate Layer (`MO` macro in QMK)
* Parameter: Layer # To Activate

In current DT defined keymap land, this would be just an entry in the keymap like `MO(LOWER)` and the corresponding "Action Type Implementation" that would handle the press and release events from that logical location.